docs/UserGuide.md

```diff
@@ -236,3 +248,17 @@ _Details coming soon ..._
 | **MarkRoomsUnclean** | `markRoomsUnclean`                                                                                                                                                                             |
 | **List**             | `list`                                                                                                                                                                                         |
 | **Help**             | `help`                                                                                                                                                                                         |
+
+--------------------------------------------------------------------------------------------------------------------
+
+## Glossary
+
+| Term             | Definition                                                                                                                               |
+|------------------|------------------------------------------------------------------------------------------------------------------------------------------|
+| **Alphanumeric** | `Refers to characters that are numbers 0-9 and alphabets A-Z, both uppercase and lowercase.`                                             |
+| **CLI**          | `Stands for Command Line Interface, it refers to a text-based interface used to execute opertaions of the system.`                       |
+| **Guest**        | `A guest staying at the hotel`                                                                                                           |
+| **GUI**          | `Stands for Graphical User Interface, it refers to an interface that allows users to interact with the system through friendly visuals.` |
+| **Hard disk**    | `A data storage device used to store and retrieve data.`                                                                                 |
+| **Index**        | `The number that corresponds to the position of the Guest in the list.`                                                                  |
+| **JSON file**    | `Stands for JavaScript Object Notation, it refers to a file format to store data.`                                                       |
```

Comment:
Other things to consider adding. Java, dd/MM/yy, parameter?

Reply:
Yes, added!